### PR TITLE
Post release fixes

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: indico/MaKaC/webinterface/tpls/ConfModificationControlFrame.tpl [hunk 1]
```diff
@@ -72,7 +72,7 @@ var methods = {'addExisting': 'event.protection.addExistingManager',
 
 var params = {confId: '${ confId }'};
 
-var modificationControlManager = new ListOfUsersManager('${ confId }',
+var modificationControlManager = new ListOfUsersManager(null,
```
Comment (ThiefMaster) [inline indico/MaKaC/webinterface/tpls/ConfModificationControlFrame.tpl]: is this also used for access rights (if the event is protected)?

File: indico/htdocs/js/indico/modules/timetable/timetable/Draw.js [hunk 2]
```diff
@@ -778,7 +778,7 @@ type("TimetableBlockNormalManagement", ["TimetableBlockNormal", "TimetableBlockM
      {
          _getTitle: function(){
              var title = this.TimetableBlockNormal.prototype._getTitle.call(this);
-             return this.eventData.friendlyId ? this.eventData.friendlyId + " - " + title : title;
+             return parseInt(this.eventData.friendlyId) >= 0 ? this.eventData.friendlyId + " - " + title : title;
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/timetable/timetable/Draw.js]: is it ever not defined? also, don't use parseInt without the second argument (`+this.eventData.friendlyId >= 0` would probably work too)

File: indico/MaKaC/webinterface/tpls/ConfModificationControlFrame.tpl (hunk 1 above)
Comment (nop33) [inline indico/MaKaC/webinterface/tpls/ConfModificationControlFrame.tpl]: Nope, `ChooseUsersPopup` is used in that case.

File: indico/htdocs/js/indico/modules/timetable/timetable/Draw.js (hunk 2 above)
Comment (nop33) [inline indico/htdocs/js/indico/modules/timetable/timetable/Draw.js]: Yeah, breaks don't have a `friendly_id`

**Comment by nop33** [comment]
included in https://github.com/indico/indico/pull/2352

**Pull request closed without merging**